### PR TITLE
Add Linting

### PR DESCRIPTION
**Pull request by kkkkv**

### CODE REVIEW

File: tgnms/.eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        - 2
+        - block
+  - env:
+      node: true
+    files:
+      - .eslintrc.js
+      - babel.config.js
+      - jest.config.js
+      - jest.*.config.js
+      - fbcnms-packages/eslint-config-fbcnms/**/*.js
```
Comment (aclave1) [inline tgnms/.eslintrc.yml:57]: we can remove every `fbcnms-packages` line that isn't part of the tgnms repo.

Comment (kkkkv) [reply tgnms/.eslintrc.yml]: removed

**Comment by kkkkv** [comment]
I added PR part

**Review by aclave1 — CHANGES_REQUESTED** [review]
create a separate workflow file and run that on PR

**Comment by aclave1** [comment]
closing in favor of #9

**Pull request closed without merging**